modules/apps/transfer/client/cli/query.go

```diff
@@ -106,3 +106,28 @@ func GetCmdParams() *cobra.Command {
 
 	return cmd
 }
+
+// GetCmdParams returns the command handler for ibc-transfer parameter querying.
+func GetCmdQueryEscrowAddress() *cobra.Command {
+	cmd := &cobra.Command{
+		Use:     "escrow-address",
+		Short:   "Get the escrow address for a channel",
+		Long:    "Get the escrow address for a channel",
+		Args:    cobra.ExactArgs(2),
+		Example: fmt.Sprintf("%s query ibc-transfer escrow-address [port] [channel-id]", version.AppName),
+		RunE: func(cmd *cobra.Command, _ []string) error {
+			clientCtx, err := client.GetClientQueryContext(cmd)
+			if err != nil {
+				return err
+			}
+			port := args[0]
+			channel := args[1]
+			addr := GetEscrowAddress(port, channel)
```

Comment:
How is this building? Shouldn't it be `types.GetEscrowAddress`?

Reply:
Ahh it isn't building

Reply:
my goodness I'm sorry about that, thanks for covering!